collector.go

```diff
@@ -185,10 +196,11 @@ func (c Collector) measure() error {
 		var measurementsPrecision []float64
 		var measurementsLeap []float64
 
-		log.Printf("WARN: clock drift is above %.2fs, taking multiple measurements for %.2f seconds", highDrift, c.NtpMeasurementDuration.Seconds())
+		log.Printf("WARN: clock drift is above %.3fs, taking multiple measurements for %.2f seconds", c.NtpHighDrift.Seconds(), c.NtpMeasurementDuration.Seconds())
 		for time.Since(begin) < c.NtpMeasurementDuration {
 			nextMeasurement, err := c.getClockOffsetAndStratum()
 			if err != nil {
+				c.serverReachable.WithLabelValues(c.NtpServer).Set(1)
```

Comment:
```suggestion
				c.serverReachable.WithLabelValues(c.NtpServer).Set(0)
```

Reply:
I put `1` there since it was able to reach the NTP server at least one time to enter inside of this loop. 
During my test, the NTP server was closing our connection since I was making too much request while looping. 

What do you think ?

Reply:
The finding is actually a pretty good reason to have 0 here which means you either need to make the NTP server more forgiving or reduce the measurement duration. 

Also we really can't have this in this branch because we could end up here when the ntp server is not reachable at all https://github.com/sapcc/ntp_exporter/blob/bc06f85152c92b9e3d78c08f63ea09aadb7f67bb/collector.go#L235

We could move this after the err != nil check but then the metric could be flappy and we definitely would want to expose this in the metrics.

Reply:
I took your advice into consideration